library/common/engine.h

```diff
@@ -85,6 +85,7 @@ class Engine {
 private:
   envoy_status_t main(std::string config, std::string log_level);
 
+  Event::Dispatcher* event_dispatcher_;
```

Comment:
I don't think this is ever getting set?

Reply:
It's returned by the dispatcher() accessor, which is used in main_interface.cc